### PR TITLE
[BUGFIX] Translate node type label in PluginView master plugin selection

### PR DESCRIPTION
**Pull request by aertmann**
Resolves: NEOS-1543

### CODE REVIEW

File: TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/ContentController.php [hunk 1]
```diff
@@ -386,7 +387,8 @@ public function masterPluginsAction(NodeInterface $node) {
 				if ($page === NULL) {
 					continue;
 				}
-				$masterPlugins[$pluginNode->getPath()] = sprintf('"%s" on page "%s"', $pluginNode->getNodeType()->getLabel(), $page->getLabel());
+				$translationHelper = new TranslationHelper();
+				$masterPlugins[$pluginNode->getIdentifier()] = sprintf('"%s" on page "%s"', $translationHelper->translate($pluginNode->getNodeType()->getLabel()), $page->getLabel());
```
Comment (dfeyer) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/ContentController.php]: Why not translating this hardcoded string at the same time ?

Comment (aertmann) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/ContentController.php]: Good point, didn't think of that. Doh

**Comment by aertmann** [comment]
:+1:

**Comment by kdambekalns** [comment]
:+1: by reading

**Comment by dfeyer** [comment]
+1 by reading

**Comment by kitsunet** [comment]
:+1:

**Pull request merged**